app/src/main/kotlin/com/simplemobiletools/dialer/adapters/SpeedDialAdapter.kt

```diff
@@ -1,20 +1,23 @@
 package com.simplemobiletools.dialer.adapters
 
+import android.view.LayoutInflater
 import android.view.Menu
 import android.view.View
 import android.view.ViewGroup
 import com.simplemobiletools.commons.adapters.MyRecyclerViewAdapter
 import com.simplemobiletools.commons.views.MyRecyclerView
 import com.simplemobiletools.dialer.R
 import com.simplemobiletools.dialer.activities.SimpleActivity
+import com.simplemobiletools.dialer.databinding.ItemSpeedDialBinding
 import com.simplemobiletools.dialer.interfaces.RemoveSpeedDialListener
 import com.simplemobiletools.dialer.models.SpeedDial
-import kotlinx.android.synthetic.main.item_speed_dial.view.speed_dial_label
 
 class SpeedDialAdapter(
     activity: SimpleActivity, var speedDialValues: List<SpeedDial>, private val removeListener: RemoveSpeedDialListener,
     recyclerView: MyRecyclerView, itemClick: (Any) -> Unit
 ) : MyRecyclerViewAdapter(activity, recyclerView, itemClick) {
+
+    private lateinit var binding : ItemSpeedDialBinding
```

Comment:
format code with ctrl + alt + L to avoid extra spaces and have proper format, also leave an empty line before init